modules/module-info.php

```diff
@@ -699,3 +699,10 @@ function jetpack_assetcdn_more_info() {
 	);
 }
 add_action( 'jetpack_module_more_info_photon-cdn', 'jetpack_assetcdn_more_info' );
+
+function jetpack_private() {
+	esc_html_e(
+		'Allows private and unlaunched sites, visible only to yourself and those you invite.'
```

Comment:
(I haven't tested this to see it in context yet, just commenting on the grammar)

I'd split this into two sentences or otherwise reword slightly. The comma reads off to me.

Reply:
Where does this render?

Reply:
Good point. I have updated it. I have no idea where it renders. Perhaps @jeherve knows?

Reply:
That data can be pulled in the Jetpack dashboard by some modals, and you could use it in your own settings if you wanted to, but right now you will not see it displayed anywhere.